share/jupyter/nbconvert/templates/gridstack/grid_type.html.j2

```diff
@@ -145,7 +145,6 @@
          data-gs-width="{{ view_data.width | default(12) }}"
          data-gs-height="{{ view_data.height | default(2) }}"
          {% if auto_position %}
-         data-gs-auto-position=true
```

Comment:
Do we want to drop support for this? or does it behave differently in gridstack 2?

Reply:
The gridstack documentation says that if x or y is missing then gridstack adds autoPosition=true.
Or that's what I understood.
"x, y - (number) element position in column/row. Note: if one is missing this will autoPosition the item"